### ISSUE TITLE
BitDepth 32  throw IndexOutOfRangeException

### ISSUE THREAD

**Issue by xxl-cc** [fo-dicom/fo-dicom#1046]
### Steps to reproduce the behavior

BitsAllocated 32 BitsStored 32 HighBit 31
 graphic.RenderImage throw  IndexOutOfRangeException on Recalculate()
the minivalue is -2147483648 
the _offset value is -minivalue  also -2147483648 
 thow System.IndexOutOfRangeException
on PrecalculatedLUT.cs Recalculate function

### fo-dicom version and OS/platform
4.05

**Comment by gofal** [comment]
Yes I am pretty sure that fo-dicom only supports 16bit grayscale. Is 32bit grayscale valid in DICOM? Can you point me to some source?
